WalletWasabi/Nostr/NostrKeyManager.cs

```diff
@@ -0,0 +1,47 @@
+using System.IO;
+using NBitcoin;
+using NBitcoin.Secp256k1;
+
+namespace WalletWasabi.Nostr;
+
+public class NostrKeyManager : IDisposable
```

Comment:
This class in unnecessary. All what it does is to make complex something that is trivial: in a config file we have to have this:

```json
{
   "AnnouncerKey": "nsec1blahblahblahblah.....",
}
```
When we create the config file the first time we create a key, that's all.

Reply:
Can you give me a hand with this? I can imagine a Json converter that reads and writes the data, and creates a key if the value on the disk doesn't exist or null.

But then how to handle the null warning in the Config file?

Reply:
Done